src/vue/forms/CreateSaleForm.vue

```diff
@@ -498,6 +498,12 @@ export default {
       return MathUtil.divide(this.form.saleInformation.hardCap,
         this.form.saleInformation.requiredBaseAssetForHardCap)
     },
+    youtubeId () {
+      const inputtedId = this.form.fullDescription.youtubeId
```

Comment:
pls rename `youtubeId` of `fullDescription` since it stores not only ID now. the same thing about `inputtedId`, it is not guaranteed that only ID link stored here

Reply:
Changed to youtubeVideo